ARCHITECTURE.md

```diff
@@ -1,292 +1,118 @@
-segmentbar\" lang=\"ja-JP\" dir=\"ltr\" data-strings=\"{ 'exit': 'ストアを離れる', 'view': '{%STOREFRONT%}ストア トップページ', 'segments': { 'smb': 'ビジネス向けストア トップページ', 'eduInd': '学生・教職員向けストア トップページ', 'other': 'ストア トップページ' } }\">\n</aside>\n<input type=\"checkbox\" id=\"ac-gn-menustate\" class=\"ac-gn-menustate\" />\n<nav id=\"ac-globalnav\" class=\"no-js\" role=\"navigation\" aria-label=\"グローバルナビゲーション\" data-hires=\"false\" data-analytics-region=\"global nav\" lang=\"ja-JP\" dir=\"ltr\" data-store-locale=\"jp\" data-store-api=\"/[storefront]/shop/bag/status\" data-search-locale=\"ja_JP\" data-search-api=\"/search-services/suggestions/\">\n\t<div class=\"ac-gn-content\">\n\t\t<ul class=\"ac-gn-header\">\n\t\t\t<li class=\"ac-gn-item ac-gn-menuicon\">\n\t\t\t\t<label class=\"ac-gn-menuicon-label\" for=\"ac-gn-menustate\" aria-hidden=\"true\">\n\t\t\t\t\t<span class=\"ac-gn-menuicon-bread ac-gn-menuicon-bread-top\">\n\t\t\t\t\t\t<span class=\"ac-gn-menuicon-bread-crust ac-gn-menuicon-bread-crust-top\"></span>\n\t\t\t\t\t</span>\n\t\t\t\t\t<span class=\"ac-gn-menuicon-bread ac-gn-menuicon-bread-bottom\">\n\t\t\t\t\t\t<span class=\"ac-gn-menuicon-bread-crust ac-gn-menuicon-bread-crust-bottom\"></span>\n\t\t\t\t\t</span>\n\t\t\t\t</label>\n\t\t\t\t<a href=\"#ac-gn-menustate\" class=\"ac-gn-menuanchor ac-gn-menuanchor-open\" id=\"ac-gn-menuanchor-open\">\n\t\t\t\t\t<span class=\"ac-gn-menuanchor-label\">メニューを開く</span>\n\t\t\t\t</a>\n\t\t\t\t<a href=\"#\" class=\"ac-gn-menuanchor ac-gn-menuanchor-close\" id=\"ac-gn-menuanchor-close\">\n\t\t\t\t\t<span class=\"ac-gn-menuanchor-label\">メニューを閉じる</span>\n\t\t\t\t</a>\n\t\t\t</li>\n\t\t\t<li class=\"ac-gn-item ac-gn-apple\">\n\t\t\t\t<a class=\"ac-gn-link ac-gn-link-apple\" href=\"/jp/\" data-analytics-title=\"apple home\" id=\"ac-gn-firstfocus-small\">\n\t\t\t\t\t<span class=\"ac-gn-link-text\">Apple</span>\n\t\t\t\t</a>\n\t\t\t</li>\n\t\t\t<li class=\"ac-gn-item ac-gn-bag ac-gn-bag-small\" id=\"ac-gn-bag-small\">\n\t\t\t\t<a class=\"ac-gn-link ac-gn-link-bag\" href=\"/jp/shop/goto/bag\" data-analytics-title=\"bag\" data-analytics-click=\"bag\" aria-label=\"ショッピングバッグ\" data-string-badge=\"商品が入っているショッピングバッグ\">\n\t\t\t\t\t<span class=\"ac-gn-link-text\">ショッピングバッグ</span>\n\t\t\t\t\t<span class=\"ac-gn-bag-badge\"></span>\n\t\t\t\t</a>\n\t\t\t\t<span class=\"ac-gn-bagview-caret ac-gn-bagview-caret-large\"></span>\n\t\t\t</li>\n\t\t</ul>\n\t\t<ul class=\"ac-gn-list\">\n\t\t\t<li class=\"ac-gn-item ac-gn-apple\">\n\t\t\t\t<a class=\"ac-gn-link ac-gn-link-apple\" href=\"/jp/\" data-analytics-title=\"apple home\" id=\"ac-gn-firstfocus\">\n\t\t\t\t\t<span class=\"ac-gn-link-text\">Apple</span>\n\t\t\t\t</a>\n\t\t\t</li>\n\t\t\t<li class=\"ac-gn-item ac-gn-item-menu ac-gn-mac\">\n\t\t\t\t<a class=\"ac-gn-link ac-gn-link-mac\" href=\"/jp/mac/\" data-analytics-title=\"mac\">\n\t\t\t\t\t<span class=\"ac-gn-link-text\">Mac</span>\n\t\t\t\t</a>\n\t\t\t</li>\n\t\t\t<li class=\"ac-gn-item ac-gn-item-menu ac-gn-ipad\">\n\t\t\t\t<a class=\"
```

Comment:
I think we should just drop this section. It’s inaccurate and out-of-date because it only matters to web-monitoring-db (from an external perspective, the structure of this database should be viewed as an implementation detail of that project, at least in the short-to-medium term). The link to the API docs, which are [mostly] up to date, are what matter here.